### ISSUE TITLE
[TW-1949] Dependencies are cropped by 1 character.

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1959]
_Martin Hohenberg on 2018-01-27T22:32:53Z says:_ 

 Hi,

I ran 

task [id] 
 
on one of my tasks that has dependencies. In this case, the last character of the precedessor is cropped away:

![Imported image](https://mail.google.com/mail/u/0/?ui=2&ik=7494b413a0&view=att&th=16139ba83d0fda41&attid=0.1&disp=safe&realattid=ii_jcxxe7iv0_16139ba82e998325&zw|width=462,height=269)

As you can see in this screenshot: The precedessor has one character more:
 
![Imported image](https://mail.google.com/mail/u/0/?ui=2&ik=7494b413a0&view=att&th=16139bb2ecec14ff&attid=0.2&disp=safe&realattid=ii_jcxxf5fg1_16139bb2fdc8a99c&zw|width=462,height=270)
This seems to happen every time, the length of the precedessor's title does not seem to have an effect, It also seems to be exactly one character.
 
I would expect the full titles to be shown in the antecessor's info.
* --[ Task Diagnostics ] ------------------

task 2.5.1 
  Platform: Linux 

Compiler 
   Version: 6.3.0 20170415 
      Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64 
Compliance: C++11 

Build Features 
     CMake: 3.7.2 
   libuuid: libuuid + uuid_unparse_lower 
 libgnutls: 3.5.6 
Build type: None 

Configuration 
      File: /home/martin/.taskrc (found), 2038 bytes, mode 100664 
      Data: /home/martin/.task (found), dir, mode 40755 
   Locking: Enabled 
        GC: Enabled 
    Server: lvps46-163-117-46.dedicated.hosteurope.de:53589 
        CA: /home/martin/.task/ca.cert.pem, readable, 3761 bytes 
     Trust: ignore hostname 
Certificate: /home/martin/.task/Martin_Hohenberg.cert.pem, readable, 3741 bytes 
       Key: /home/martin/.task/Martin_Hohenberg.key.pem, readable, 25364 bytes 
   Ciphers: NORMAL 
     Creds: PUBLIC/Martin Hohenberg/************************************ 

Hooks 
    System: Enabled 
  Location: /home/martin/.task/hooks 
    Active: on-modify.timewarrior (executable) 
  Inactive:  

Tests 
     $TERM: xterm-256color (255x53) 
      Dups: Scanned 264 tasks for duplicate UUIDs: 
            No duplicates found 
Broken ref: Scanned 264 tasks for broken references: 
            No broken references found

I know this seems like a minor issue, but still, even small bugs may lead to unintended consequences later-on. 
 
Kind regards, 
 
Martin

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2018-01-27T22:32:53Z
Modified: 2018-01-28T20:09:57Z
```

**Comment by taskwarrior** [comment]
_Martin Hohenberg on 2018-01-28T12:13:45Z says:_ 

 In further testing, it appears this only happens when I have Umlaute (or other non-ASCII-127 characters) in the precedessor's title. 

This may be an Unicode issue.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2018-01-28T20:07:51Z says:_ 

 I believe this is fixed already in 2.6.0.

**Comment by pbeckingham** [comment]
Fixed in 2.6.0+
